app/common/typed-ipc.ts

```diff
@@ -63,7 +63,7 @@ export type RendererMessage = {
   "set-idle": () => void;
   "show-keyboard-shortcuts": () => void;
   "show-notification-settings": () => void;
-  "switch-server-tab": (index: number) => void;
+  "switch-server-tab": (id: string) => void;
```

Comment:
Is this a server ID or a tab ID? (Are those two different things?)

It'd be good to make the answer explicit: use specific names like `serverId` or `tabId`, rather than just `id`, everywhere except the ID property that's right on the object it identifies. (And then if it seems most convenient to use the same name there too, that's also a popular choice.)

Reply:
> Is this a server ID or a tab ID? (Are those two different things?)

We set tab ID to server ID in the app for server tabs, but not the other way around. For functional tabs, tabId is just a unique string ID. 

tabId would be more appropriate to use here, since we are talking about switching tabs.